modules/tcp_performConnection/performConnection.c

```diff
@@ -70,8 +69,8 @@ int performConnection(int sockfd) {
 
   // 2. Send client version
   const char *client_version = "VERSION 2.42\n";
-  if (sendMessage(sockfd, client_version) != EXIT_SUCCESS) {
-    return EXIT_FAILURE;
+  if (sendMessage(sockfd, client_version) != 0) {
+    return -1;
```

Comment:
```suggestion
    return EXIT_FAILURE;
```
We use EXIT_FAILURE & EXIT_SUCCESS as  standard